### PR TITLE
Add details to pjsip

### PR DESCRIPTION
**Pull request by KristaHyer**
My autoformatting on VSCode has changed the formatting in some cases, this wasn't deliberate but I intend to leave it.

Tested by pointing sprout at this version for its submodule of pjsip, and running `make full_test`, which passed.

### CODE REVIEW

File: README.txt [hunk 1]
```diff
@@ -1,3 +1,7 @@
+ This version of pjsip has been forked off r4448, found at
+ http://svn.pjsip.org/repos/pjproject/trunk/, and modified slightly by
+ Metaswitch.
```
Comment (tomwatson1024) [inline README.txt]: Do we need to specify a time period here?

File: pjlib-util/src/pjlib-util/xml.c [hunk 2]
```diff
@@ -1,5 +1,10 @@
+/**
```
Comment (tomwatson1024) [inline pjlib-util/src/pjlib-util/xml.c]: One `*` would be more consistent with the rest of the file.

File: pjlib/include/pj/compat/socket.h [hunk 3]
```diff
@@ -1,3 +1,8 @@
+/**
```
Comment (tomwatson1024) [inline pjlib/include/pj/compat/socket.h]: One star again - I won't comment on this from here on, as I'm not paid by the comment 😛

File: pjlib/include/pj/compat/socket.h [hunk 4]
```diff
@@ -1,3 +1,8 @@
...
+ * Some of the content of this file has been edited by Metaswitch, in the time
+ * period from May 2013 to the present time.
+*/
```
Comment (tomwatson1024) [inline pjlib/include/pj/compat/socket.h]: Starting this line with a space would keep the `*`s in line.

File: pjlib/include/pj/string.h [hunk 5]
```diff
@@ -1,3 +1,8 @@
+/**
+ * Some of the content of this file has been edited by Metaswitch, in the time
+ * period from May 2013 to the present time.
+*/
```
Comment (tomwatson1024) [inline pjlib/include/pj/string.h]: Same comment about starting with a space.

File: pjlib/src/pj/activesock.c [hunk 6]
```diff
@@ -1,3 +1,8 @@
+/**
+ * Some of the content of this file has been edited by Metaswitch, in the time
+ * period from May 2013 to the present time.
+*/
```
Comment (tomwatson1024) [inline pjlib/src/pj/activesock.c]: Spaces, as per.

File: pjlib/include/pj/sock.h [hunk 7]
```diff
@@ -277,7 +282,7 @@ extern const pj_uint16_t PJ_IPTOS_MINCOST;
 
 
 /**
```
Comment (tomwatson1024) [inline pjlib/include/pj/sock.h]: Looks like pjsip isn't consistent with its commenting style between files. It's probably still better to match the style of the file though.

File: README.txt (hunk 1 above)
Comment (KristaHyer) [reply README.txt]: It's listed elsewhere (52 times!), so I plan to leave it out here 😄

Comment (tomwatson1024) [reply README.txt]: Fair enough 😺

**Pull request merged**